engine/src/test/java/io/graphqlcrud/GraphQLSchemaBuilderTest.java

```diff
@@ -59,6 +59,59 @@ public void testSchemaPrint() throws Exception {
             Assertions.assertNotNull(accountsDef.getDirective("sql"));
             Assertions.assertEquals("[SSN]", accountsDef.getDirective("sql").getArgument("keys").getValue().toString());
             Assertions.assertEquals("[SSN]", accountsDef.getDirective("sql").getArgument("reference_keys").getValue().toString());
+
+            GraphQLFieldDefinition fieldDefinition = objectType.getFieldDefinition("customers");
+            Assertions.assertEquals("page",fieldDefinition.getArguments().get(0).getName());
+            Assertions.assertEquals("filter", fieldDefinition.getArguments().get(1).getName());
+
+            schema.getTypeMap().entrySet().forEach(parent -> {
+                if(parent.getKey().equals("SortDirectionEnum")) {
+                    Assertions.assertEquals("SortDirectionEnum",parent.getValue().getName());
+                }
+                else if(parent.getKey().equals("OrderByInput")) {
+                    Assertions.assertEquals("OrderByInput",parent.getValue().getName());
+                    parent.getValue().getChildren().forEach(child -> {
+                        if(child.toString().contains("name=field"))
+                            Assertions.assertTrue(child.toString().contains("originalType=String!"));
+                        else if(child.toString().contains("name=order"))
+                            Assertions.assertTrue(child.toString().contains("defaultValue=ASC"));
+                    });
+                }
+                else if(parent.getKey().equals("PageRequest")) {
+                    parent.getValue().getChildren().forEach(child -> {
+                        if(child.toString().contains("name=limit"))
+                            Assertions.assertTrue(child.toString().contains("name=Int"));
+                        else if(child.toString().contains("name=offset"))
+                            Assertions.assertTrue(child.toString().contains("name=Int"));
+                    });
+                }
+                else if(parent.getKey().equals("QueryFilter")) {
+                    Assertions.assertEquals("QueryFilter",parent.getValue().getName());
+                    parent.getValue().getChildren().forEach(child -> {
+                        if(child.toString().contains("name=and"))
+                            Assertions.assertTrue(child.toString().contains("originalType=[QueryFilter]"));
+                    });
+                }
+                else if(parent.getKey().equals("FloatInput")) {
```

Comment:
Expand the test to each individual input type you added. Break them into different methods be more readable.